frontend/src/styles/ProjectsView.css

```diff
@@ -0,0 +1,60 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.nav-hardwarelogo {
+    font-size: 1.5rem;
+    color: white;
+    text-decoration: none;
+    background-color: orange;
+}
+
+.nav-links {
+    list-style: none;
+    background-color: orange;
+    display: flex;
+    gap: 1.5rem;
+}
+
+.nav-links li {
+    background-color: orange;
+    font-size: 1rem;
+}
+
+.nav-links a {
+    background-color: orange;
+    color: white;
+    text-decoration: none;
+}
+
+.nav-links a:hover {
+    color: white;
+}
+
+.nav-hardwarelogo:hover {
+    color: white;
+}
+
+.create-new-proj {
+    padding: 5rem;
+    width: 50%;
+}
+
+.create-new-proj a {
+    text-align: center;
+}
+
+.project-list-div {
```

Comment:
No hungarian notion.

Reply:
This is not me this is Alec's code for his project list css